bundles/org.openhab.binding.automower/README.md

```diff
@@ -13,7 +13,7 @@ All Husqvarna Automower models with "Automower Connect" should be supported. It
 
 ## Discovery
 
-Once the bridge is created and configured, openHAB will automatically discover all Automowers registered on your account.
+Once the bridge is created and configured, OpenHab will automatically discover all Automowers registered on your account.
```

Comment:
Please revert 
```suggestion
Once the bridge is created and configured, openHab will automatically discover all Automowers registered on your account.
```

Reply:
did it. was also wondering where this came from - was thinking about spotless:apply

Reply:
Still needs a fix, it is `openHAB` not `openHab`

Reply:
should be fine now. sorry.